keyboards/lily58/keymaps/spanish_latam/keymap.c

```diff
@@ -0,0 +1,154 @@
+#include QMK_KEYBOARD_H
+
+enum layer_number {
+  _QWERTY = 0,
```

Comment:
0 isn't needed here as it starts from 0 by default. 
```suggestion
  _QWERTY,
```